(#4) 로그인 페이지 마크업

dlatns0201 opened the pull request:
## 이슈
#4 로그인 작성

## 참고사항
1. `pages/Login/index.tsx` 에서 Form 영역을 별도의 컴포넌트로 분리할 예정
2. css module 변수명은 `Style`로 통일
3. label이 위에 위치한 Input을 공통 컴포넌트 `components/Input/BasicInput` 으로 작성
   - 아직 불안정한 컴포넌트라 손봐야할 필요가 있음
4. Input 아래에 있는 메세지는 Validate를 작성하면서 미노출 설정할 예정
5. antd 컴포넌트에 스타일을 주기 위해 css module 클래스이름을 줄 땐 두 개이상의 Nested 클래스를 주지 않으면 스타일이 적용되지 않는 경우가 많음
   - 그럴 때 Nested ClassName을 `.override` 로 명명
   ```scss
   .btn_submit {
     &.override {
       display: block;
       height: auto;
       margin: 60px auto 0;
       padding: 18px 131px;
       background: #ffd153;
       border-radius: 27px;
       font-size: 18px;
       line-height: 22px;
       color: #000;
       box-shadow: 0 3px 6px 0 rgba(0, 0, 0, 0.16);
     }
   }
   ```
   ```jsx
   <BasicInput
       type="submit"
       label="로그인"
       className={cx(Style.btn_submit, Style.override)}
   />
   ```

## 스크린샷
![image](https://user-images.githubusercontent.com/40534721/103229663-db59dc00-4976-11eb-83fb-d95d1b02b484.png)

dlatns0201 committed:
feat(login): (#4) 라우터 및 public 레이아웃 연결

dlatns0201 committed:
fix: 빌드 스크립트 작성 및 리덕스 devtool 설정 에러 처리

dlatns0201 committed:
chore(package): (#4) antd 및 classnames 추가

dlatns0201 committed:
chore(eslint): (#4) eslint 규칙 추가

dlatns0201 committed:
style: (#4) normalize css 작성 및 antd css 추가

dlatns0201 committed:
feat(markup): (#4) Public Layout 및 Nav 작성

로그인, 회원가입처럼 인증 없이 사용하는 layout 규격 작성

dlatns0201 committed:
feat: (#4) 기본 Form Input 컴포넌트 작성

추후에 Submit 버튼을 분리할 가능성 존재

dlatns0201 committed:
feat: (#4) 로그인 페이지 마크업

비즈니스 로직을 작성하면서 페이지 내의 폼을 컴포넌트로 빼낼 예정

dlatns0201 committed:
fix: (#4) css module 변수명 변경에 따른 작업

SaraHan774 commented on src/pages/Login/index.tsx:
```diff
@@ -0,0 +1,44 @@
+        <div className={Style.form_container}>
+          <Form name="user">
+            <BasicInput label="이메일" name="email" message="이메일을 입력해주세요" />
+            <BasicInput label="비밀번호" name="password" message="비밀번호를 입력해주세요" />
```
name="password" 에서 type 으로 바꿔야할듯 합니다

dlatns0201 replied:
name은 그대로 가는 것이 맞고 컴포넌트 호출 속성으로 type을 뺐었습니다 ㅎㅎ

dlatns0201 committed:
chore: #4 style 관련 세팅 수정

path alias 조정
common.scss
index.scss 제거
global.scss 생성 후 app.scss 에서 전역 스타일 가져오기

dlatns0201 committed:
style: #4 css module 방식에서 BEM 기반 설계의 scss 로 변경

dlatns0201 committed:
chore: #4 path alias 재설정

dlatns0201 committed:
fix: #4 폼 전송버튼, Basic Input 에서 분리

dlatns0201 committed:
fix: #4 Basic Input 대신 SubmitBtn 적용

dlatns0201 committed:
fix(basic-input) #4 BasicInput 필요외의 속성 제거 및 type 속성 추가

dlatns0201 committed:
style(page-login): #4 아이디 찾기 링크 float 제거

dlatns0201 committed:
fix(public-layout): #4 public layout 스타일 제거
